stdlib/listLabels.mli

```diff
@@ -103,9 +103,8 @@ val init : len:int -> f:(int -> 'a) -> 'a list
  *)
 
 val append : 'a list -> 'a list -> 'a list
-(** Concatenate two lists. Same function as the infix operator [@].
-   Not tail-recursive (length of the first argument). The [@]
-   operator is not tail-recursive either.
+(** [append l0 l1] appends [l1] to [l0].
+     Same function as the infix operator [@].
  *)
 
 val rev_append : 'a list -> 'a list -> 'a list
```

Comment:
I guess the documentation of `rev_append` should also be changed to remove "but is tail-recursive", as was done in #11362

Reply:
Well spotted!  Fixed in [f03c278](https://github.com/ocaml/ocaml/pull/11859/commits/f03c278d0dfe87705880b5eb6b75f1ce6598c46a).